lib/src/react16_suggestors/react_style_maps_updater.dart

```diff
@@ -39,36 +40,40 @@ class ReactStyleMapsUpdater extends GeneralizingAstVisitor
 
           /// CSS properties that were modified by the script or need to be
           /// manually checked
-          List<String> affectedValues = [];
+          List<String> listOfVariables = [];
 
           // Variables that affect the writing of the comment that goes above
           // the style prop. There are different messages patched based on
           // these booleans.
           bool styleMapContainsAVariable = false;
           bool isAVariable = false;
           bool isAFunction = false;
+          bool isUnknown = false;
```

Comment:
Can you add a quick test case for this? I think a prefixed identifier would hit it:

```
..style = {'width': foo.bar}
```

Reply:
#nit Now that I think about it, maybe this should be called `isOther`... 😅

Reply:
Good call!